third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Runtime Configurator"
+layout: "google"
+page_title: "Google: google_runtimeconfig_config"
+sidebar_current: "docs-google-datasource-runtimeconfig-config"
+description: |-
+  Get information about a Google Cloud RuntimeConfig.
+---
+
+# google\_runtimeconfig\_config
+
+To get more information about RuntimeConfigs, see:
+
+* [API documentation](https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs)
+* How-to Guides
+    * [Runtime Configurator Fundamentals](https://cloud.google.com/deployment-manager/runtime-configurator/)
+
+## Example Usage
+
+```hcl
+data "google_runtimeconfig_config" "run-service" {
+  name = "my-service"
+  description = "foobat"
```

Comment:
```suggestion
  description = "foobar"
```
We could just remove `description` entirely right? It wouldn't help identify the resource

Reply:
fixed